src/Umbraco.Web.UI.Client/src/installer/steps/user.controller.js

```diff
@@ -58,6 +58,15 @@ angular.module("umbraco.install").controller("Umbraco.Install.UserController", f
 
       const pips = consentSlider.querySelectorAll('.noUi-value');
 
+      let activePip = [null, null];
+      consentSlider.noUiSlider.on('update', function (values,handle) {
+        if(activePip[handle]){
+          activePip[handle].classList.remove("noUi-value-active");
+        }
+        activePip[handle] = pips[parseInt(values[handle])-1];
+        activePip[handle].classList.add("noUi-value-active");
+      });
+
```

Comment:
I don't think we need this if the class `noUi-value-active` is handle inside the `<umb-range-slider>` component.

Reply:
I'll leave this alone for now, until there is some clarification as to whether the installation steps should use the range-slider component instead

Reply:
Yes, currently we can't use many components outside backoffice, which it why it doesn't use `<umb-range-slider>` here:

But it would be great to update the logic in the two parts to be consistent:

```
const dataValue = Math.round(values[handle]);

activePip[handle] = sliderInstance.querySelector('.noUi-value[data-value="' + dataValue + '"]');
```

I think we should test this with a slider which use decimal values.

Reply:
I'm not sure it would work to round a decimal and then query that rounded number. But if it doesn't the  skipping the rounding step may be fine